webapp/packages/core-blocks/src/FormControls/useForm.ts

```diff
@@ -31,6 +32,7 @@ export function useForm(options?: IOptions): IFormContext {
   }
 
   const disableEnterSubmit = options?.disableEnterSubmit ?? parentForm?.disableEnterSubmit ?? false;
+  const disableCtrlEnterSubmit = options?.disableCtrlEnterSubmit ?? parentForm?.disableCtrlEnterSubmit ?? false;
```

Comment:
i'm not sure that we need special prop for that (so we can use only `disableEnterSubmit`)